communications/views.py

```
@@ -264,56 +264,68 @@ def africas_talking_voice_callback(request):
-            <Say voice="woman">La ilaha illa Allah.</Say>
-            <Say voice="woman">May Allah accept your prayers.</Say>
+            <Say voice="woman">{message}</Say>
         </Response>'''
-    
+
+        else:
+            # Unknown call type
+            xml_response = f'''<?xml version="1.0" encoding="UTF-8"?>
+        <Response>
+            <Say voice="woman">This is a test call from Muadhin.</Say>
+        </Response>'''
```

Ruff (0.14.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)